keyboards/lily58/keymaps/oldfieldtc/keymap.c

```diff
@@ -0,0 +1,203 @@
+#include QMK_KEYBOARD_H
```

Comment:
Please add an appropriate [copyright header](https://docs.qmk.fm/#/pr_checklist?id=example-gplv2-header) here:
```suggestion
// Copyright 2022 Tommy Oldfield (@oldfieldtc)
// SPDX-License-Identifier: GPL-2.0-or-later

#include QMK_KEYBOARD_H
```